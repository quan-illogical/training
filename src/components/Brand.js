import React from 'react'
import logo from "../images/brand-logo.svg"

export default function Brand() {
    return (
        <div className="brand-component">
            <img src={logo} alt="Logo"/>
            <b>START YOUR PERSONAL PHOTO EXPERIENCE</b>
            <h5>Login Your Account</h5>
        </div>
    )
}
