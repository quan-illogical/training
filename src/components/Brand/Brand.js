import React from 'react'
import logo from "../../images/brand-logo.svg"

export default function Brand(props) {
    return (
        <div className="brand-component">
            <img src={logo} alt="Logo"/>
            <b>START YOUR PERSONAL PHOTO EXPERIENCE</b>
            <h5>{props.title || "Login Your Account"}</h5>
        </div>
    )
}
