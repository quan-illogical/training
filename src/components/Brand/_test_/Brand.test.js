import React from 'react'
import { shallow } from 'enzyme'
import Brand from "../Brand"
import logo from "../../../images/brand-logo.svg"


describe("Brand", () => {
    it("should render Brand correctly", ()=>{
        const wrapper = shallow(<Brand/>)
        expect(wrapper).toMatchSnapshot()
    })

    it("should render title", () => {
        const wrapper = shallow(<Brand/>)
        const title = wrapper.find("h5")
        expect(title).toHaveLength(1)
        expect(title.text()).toEqual('Login Your Account')
    })

    it("should render image of the company logo", () => {
        const wrapper = shallow(<Brand/>)
        expect(wrapper.find("img").prop("src")).toEqual(logo)
    })
})