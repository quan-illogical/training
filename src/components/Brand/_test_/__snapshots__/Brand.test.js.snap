// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand should render Brand correctly 1`] = `
<div
  className="brand-component"
>
  <img
    alt="Logo"
    src="brand-logo.svg"
  />
  <b>
    START YOUR PERSONAL PHOTO EXPERIENCE
  </b>
  <h5>
    Login Your Account
  </h5>
</div>
`;
